### DISCUSSION TITLE
Published an article about Gidon - Avalonia based new MVVM IoC plugin container

### DISCUSSION THREAD

**Discussion by npolyak** [AvaloniaUI/Avalonia#7672] [General]
Published an article about Gidon - an Avalonia based MVVM IoC Plugin  framework that I am working on - [Gidon - Avalonia based MVVM Plugin IoC Container](https://www.codeproject.com/Articles/5325733/Gidon-Avalonia-based-MVVM-Plugin-IoC-Container).

**Comment by Sorien** [comment]
nice article, 

just a small notice there is a missing image
![image](https://user-images.githubusercontent.com/1270275/155101787-4717bc03-9054-400e-b315-09191820ab22.png)

**Reply by npolyak** [reply]
I just fixed it. Thanks. If you liked the article, please vote for it.
